README.md

```diff
@@ -1,73 +1,44 @@
 ![Build Status](https://storage.googleapis.com/cloud-tools-for-java-kokoro-build-badges/intellij-ubuntu-master-eap-orb.svg)
 
-The plugin integrates the [Google Cloud Platform](https://cloud.google.com/)
-into the IntelliJ IDEA UI. Currently this includes:
+Cloud Tools for IntelliJ is a plugin that helps facilitates cloud development. This includes the 
+following areas:
+
+* [Kubernetes](https://github.com/GoogleCloudPlatform/google-cloud-intellij/tree/master/kubernetes)
+* Google Cloud Platform
+  * [Google Cloud Source Repositories](https://cloud.google.com/tools/cloud-repositories/) 
+    Fully-featured private Git repositories hosted on Google Cloud Platform.
+  * The [Google Cloud Debugger](https://cloud.google.com/tools/cloud-debugger/) 
+    The Cloud Debugger can inspect the state of a Java application running on 
+    [Google App Engine](https://cloud.google.com/appengine/)
+    at any code location without stopping the application.
+  * [Google App Engine](https://cloud.google.com/appengine/docs/) deployment via the Cloud SDK.
+  * [Google Cloud Storage](https://cloud.google.com/storage/) 
+    Browse your Google Cloud Storage buckets.
+  * [Google Cloud Java Client Libraries](https://cloud.google.com/tools/intellij/docs/client-libraries) 
+    Add Java client libraries to your project, enable Google Cloud APIs, and create service accounts.
+
+    (_For detailed user documentation on GCP features, visit our documentation_
+ [website](https://cloud.google.com/tools/intellij/docs/?utm_source=github&utm_medium=google-cloud-intellij&utm_campaign=ToolsforIntelliJ)).
 
-* [Google Cloud Source Repositories](https://cloud.google.com/tools/cloud-repositories/) 
-  Fully-featured private Git repositories hosted on Google Cloud Platform.
-* The [Google Cloud Debugger](https://cloud.google.com/tools/cloud-debugger/) 
-  The Cloud Debugger can inspect the state of a Java application running on 
-  [Google App Engine](https://cloud.google.com/appengine/)
-  at any code location without stopping the application.
-* [Google App Engine](https://cloud.google.com/appengine/docs/) deployment via the Cloud SDK.
-* [Google Cloud Storage](https://cloud.google.com/storage/) 
-  Browse your Google Cloud Storage buckets.
-* [Google Cloud Java Client Libraries](https://cloud.google.com/tools/intellij/docs/client-libraries) 
-  Add Java client libraries to your project, enable Google Cloud APIs, and create service accounts.
+## Supported Platforms
 
-For detailed user documentation go to our documentation
- [website](https://cloud.google.com/tools/intellij/docs/?utm_source=github&utm_medium=google-cloud-intellij&utm_campaign=ToolsforIntelliJ).
+The Cloud Tools for IntelliJ plugin supports the entire JetBrains family of IDEs, versions 2018.2 or 
+later for Kubernetes support, and 2017.2 for everything else. Both Community and Ultimate editions
```

Comment:
SInce we support multiple IDEs, should we rephrase more generally, i.e. *both free and paid editions of the JetBrains family...*?